source/leuven/services/openondemand.rst

```diff
@@ -413,6 +413,28 @@ You can now start working in your own customized environment.
 
 For more general information, please refer to the `official JupyterLab documentation`_.
 
+Exporting Jupyter Notebooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The native file format for jupyter notebooks is ``.ipynb``, which is already portable.
+However, you may want to convert your notebook to other interesting formats.
+For a correct working of this feature, choose 'Convert notebook to ... formats' on the form.
+Among different possible conversion formats, the following are interesting:
+
+- ``.py`` (to execute it directly as a Python script)
+- ``.md`` (to add to reports or Quarto slides)
+- ``.rst`` (for internal documentation or reports in reStructuredText format)
+- ``.html`` (for blog posts)
+- ``.pdf`` (as a portable document)
+
+For that, you need to go to the 'Files' menu and from the 'Save and Export Notebook As'
+option, and eventually choose your desired format.
+
+However, to convert specifically to PDF format, you need to take additional steps.
+First, save the notebook in your current folder as e.g. ``<notebook>.tex`` by choosing ``LaTeX``.
+Then, from the Terminal on JupyterLab, execute ``xelatex <notebook>.tex``.
+This will produce ``<notebook>.pdf`` in the current folder.
```

Comment:
I think we better shrink this down to the essentials by referring to the Jupyter documentation as much as possible. Also, I suppose users may need to load a module in order to have access  to ``xelatex``. 

 ```suggestion
In addition to the native ``.ipynb`` format, Jupyter notebooks can be [exported to various other formats](
https://jupyterlab.readthedocs.io/en/stable/user/export.html#exporting-notebooks).
Directly exporting to PDF will however not work in this app. Instead, first export
the notebook in ``LaTex`` format (to for example a ``notebook.tex`` file) and then execute
``xelatex notebook.tex`` in a terminal, which will produce the corresponding PDF file.
You may need to first load a ``texlive`` module in order to get hold of ``xelatex``.
```